astropy/stats/jackknife.py

```diff
@@ -0,0 +1,194 @@
+# Licensed under a 3-clause BSD style license - see LICENSE.rst
+
+from __future__ import (absolute_import, division, print_function,
+                        unicode_literals)
+import numpy as np
+from scipy.special import erfinv
```

Comment:
@mirca - The tests are still failing with the missing scipy as the import is wrongly placed. Since the ``jackknife`` submodule is imported in the ``__init__.py``, no optional deps import should be in the toplevel of the submodul, but only at the top of the affected functions.